tests/cards/ares/DeimosDownAres.spec.ts

```diff
@@ -104,18 +104,21 @@ describe('DeimosDownAres', function() {
     adjacentSpaces[0].bonus = []; // Just in case it had steel bonuses
     game.addGreenery(player, adjacentSpaces[0]);
     runAllActions(game);
-    expect(player.megaCredits).to.eq(1); // 1MC for Owner Adjacent
+
+    expect(player.megaCredits).to.eq(1); // Owner gets MC bonus
+
+    // Player 1 gets adjacency bonuses
     expect(player.steel).to.eq(1); // 1 Steel for adjacency bonus
-    expect(adsPlayer1.resourceCount).to.eq(1); // 1 Asteroid
+    expect(player.getResourceCount(CardResource.ASTEROID)).eq(1);
```

Comment:
Just to make testing a little simpler, I used `getResourceCount` instead of having to keep track of each card.